Modules/VMware.Hv.Helper/VMware.HV.Helper.psm1

```diff
@@ -4460,6 +4460,7 @@ function New-HVPool {
           $maximumCount = $jsonObject.AutomatedDesktopSpec.VmNamingSpec.patternNamingSettings.maxNumberOfMachines
           $spareCount = $jsonObject.AutomatedDesktopSpec.VmNamingSpec.patternNamingSettings.numberOfSpareMachines
           $provisioningTime = $jsonObject.AutomatedDesktopSpec.VmNamingSpec.patternNamingSettings.provisioningTime
+          if ($provisioningTime -eq 'ON_DEMAND') { $MinimumCount = $jsonObject.AutomatedDesktopSpec.VmNamingSpec.patternNamingSettings.MinNumberOfMachines }
```

Comment:
It looks like setting $minimumcount from the json spec was omitted and you're now adding that. But why adding it only if the $provisioningTime is "ON_DEMAND"? Does it only make sense in this case?

Reply:
Yes according to [this help](https://vdc-repo.vmware.com/vmwb-repository/dcr-public/2365e8a2-2a49-40c3-ae46-510d918c2fa1/86751889-022e-47c4-b2b9-8bc2e3fa6d21/vdi.resources.Desktop.PatternNamingSettings.html) from VMware it does make sense only if `$provisioningTime` is `ON_DEMAND`. It is also a security check because if `$provisioningTime` is `UP_FRONT`, `MinNumberOfMachines` may not be set or $null